### PR TITLE
samples: update slates and CDN keys to return LROs; add live config s…

### PR DESCRIPTION
**Pull request by irataxy**
…amples and tests

b: 277359252

Please merge on approval

### CODE REVIEW

**Review by bajajneha27 — APPROVED** [review]

File: google-cloud-video-stitcher/samples/create_cdn_key.rb [hunk 1]
```diff
@@ -60,9 +60,14 @@ def create_cdn_key project_id:, location:, cdn_key_id:, hostname:, key_name:, pr
                   }
                 end
 
-  response = client.create_cdn_key parent: parent, cdn_key: new_cdn_key, cdn_key_id: cdn_key_id
+  operation = client.create_cdn_key parent: parent, cdn_key: new_cdn_key, cdn_key_id: cdn_key_id
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/create_cdn_key.rb:63]: I recommend keeping lines shorter than 80 columns, to ensure reasonable formatting when these appear on cloudsite. (Here and a number of additional places below.) Doesn't matter so much for tests above, but for the samples themselves, I think we should try to be be pretty strict about it.

File: google-cloud-video-stitcher/samples/create_live_config.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+##
+# Create a live config
+#
+# @param project_id [String] Your Google Cloud project (e.g. "my-project")
+# @param location [String] The location (e.g. "us-central1")
+# @param live_config_id [String] Your live config name (e.g. "my-live-config")
+# @param source_uri [String] Uri of the live stream to stitch
+#                            (e.g. "https://storage.googleapis.com/my-bucket/main.mpd")
+# @param ad_tag_uri [String] Uri of the ad tag (e.g. "https://pubads.g.doubleclick.net/gampad/ads...")
+# @param slate_id [String] The default slate ID to use when no slates are specified in an ad break's message
```
Comment (dazuma) [inline google-cloud-video-stitcher/samples/create_live_config.rb:27]: Some of these lines are quite long.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-video-stitcher/samples/create_cdn_key.rb (hunk 1 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/create_cdn_key.rb]: Done

File: google-cloud-video-stitcher/samples/create_live_config.rb (hunk 2 above)
Comment (irataxy) [reply google-cloud-video-stitcher/samples/create_live_config.rb]: applied 80-column limit (where possible)

**Pull request merged**